operations/research-and-development/product/growth/pql-definition.md

```diff
@@ -0,0 +1,9 @@
+### v1 PQL definition as of May 10, 2022
```

Comment:
Is it still the current definition?

Reply:
Yes. Updating definition currently blocked by ability to view PQL -> opportunity data in SFDC. See [here](https://docs.google.com/document/d/1y_qSw7YEPtD_YQq19_BsV0eBl_Hngl-eQSeQu7WV2cE/edit#heading=h.mqhwsdw08d60) for details

Reply:
@johndavidlugtu propose adding today's date, in that case?